src/ui/components/Notice/styles.scss

```diff
@@ -10,6 +10,12 @@
   line-height: 1.1;
   padding: 4px;
 
+  &.Notice-light {
```

Comment:
I think you should also remove the padding for `light` notices. Since it doesn't have a colorful container, I think the padding is not helpful. In this case, it expands the margin which looks inconsistent with all other margins:

<img width="358" alt="screenshot 2018-09-25 14 18 40" src="https://user-images.githubusercontent.com/55398/46037564-18ff9880-c0ce-11e8-9a60-68c505bc96e3.png">